src/app/games/games.service.ts

```diff
@@ -85,6 +85,11 @@ export class GamesService {
         });
         return this.http.request(request).do(() => {
             this.toastyService.success('Player added!');
+        }).catch(err => {
+            if (err.status === 409) {
+                this.toastyService.warning('Player is already in game');
```

Comment:
What we truly need is some way to prevent that from happening in the first place. Like if you want someone to invite and they already invited you see it